File: Queue/Queue.c
#include "stdio.h"
#include "stdlib.h"

struct Node
{
    int data;
    struct Node *next;
};

struct Node *front = NULL;
struct Node *rear = NULL;

void Display(){
    struct Node *ptr;
    ptr = front;
    while (ptr!=NULL){
        printf("%d ", ptr->data);
        ptr = ptr->next;
    }
}

void enqueue(int val)
{
    struct Node *newNode;
    newNode = (struct Node *)malloc(sizeof(struct Node));
    if (newNode == NULL)
        printf("Queue is full!\n");
    else
    {
        newNode->data = val;
        newNode->next = NULL;
        if (front == NULL)
            front = rear = newNode;
        else
        {
            rear->next = newNode;
            rear = newNode;
        }
    }
}

int dequeue()
{
    int x = -1;
    if (front == NULL)
        printf("Queue is empty!");
    else{
        x = front->data;
        struct Node *ptr = front;
        front = front->next;
        free(ptr);
    }
    return x;
}

int main()
{
    enqueue(1);
    enqueue(2);
    enqueue(3);
    enqueue(4);
    enqueue(5);
    dequeue();
    dequeue();
    Display();
}

File: Queue/Queue.py

class Node:
    def __init__(self,val):
        self.data = val
        self.next = None
        
class Queue:
    def __init__(self):
        self.front = None
        self.rear = None
        
    def Display(self):
        ptr = self.front
        while ptr:
            print(ptr.data, end=" ")
            ptr = ptr.next
        print()
        
    def enqueue(self, val):
        newNode = Node(val)
        if not newNode:
            print("Queue is full!")
        else:
            newNode.next = None
            if self.front==None:
                self.front = self.rear = newNode
            else:
                self.rear.next = newNode
                self.rear = newNode
    
    def dequeue(self):
        x = -1
        if self.front==None:
            print("Queue is empty!")
        else:
            x = self.front.data
            self.front = self.front.next
        return x
    
if __name__=='__main__':
    q = Queue()
    q.enqueue(1)
    q.enqueue(2)
    q.enqueue(3)
    q.enqueue(4)
    q.enqueue(5)
    q.dequeue()
    q.dequeue()
    q.Display()